rclcpp/include/rclcpp/node.hpp

```diff
@@ -392,6 +392,14 @@ class Node : public std::enable_shared_from_this<Node>
   std::vector<std::string>
   get_node_names() const;
 
+  RCLCPP_PUBLIC
+  std::vector<std::string>
+  get_node_namespaces() const;
+
+  RCLCPP_PUBLIC
+  std::vector<std::string>
+  get_node_names_namespaces_formatted(const char & separator) const;
```

Comment:
I think this method isn't really necessary, since the separator is know (`/`). It might change in the future, but that's nothing that user should need to care about.

Reply:
I've removed this method, and changed `get_node_names` to return the fully qualified name.